server/latest.go

```diff
@@ -0,0 +1,258 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package server
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/aws/client"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/aws-sdk-go/service/s3/s3manager"
+	"github.com/pkg/errors"
+)
+
+// Get the existing wensite config if any exists
+func getBucketConfig(svc *s3.S3, bucket string) (*s3.GetBucketWebsiteOutput, error) {
+
+	input := &s3.GetBucketWebsiteInput{
+		Bucket: aws.String(bucket),
+	}
+
+	result, err := svc.GetBucketWebsite(input)
+	if err != nil {
+		if err, ok := err.(awserr.Error); ok {
+			switch err.Code() {
+			default:
+				LogError(err.Error())
+			}
+		} else {
+			LogError(err.Error())
+		}
+		return nil, err
+	}
+	return result, nil
+}
+
+func checkBucket(svc *s3.S3, input *s3.ListObjectsV2Input) (*s3.ListObjectsV2Output, error) {
+	result, err := svc.ListObjectsV2(input)
+	return result, err
+}
+
+func checkIfBucketExistsWithPrefixAndWait(ctx context.Context, svc *s3.S3, cfg *MatterbuildConfig, ver string, typeToRelease string) (*s3.ListObjectsV2Output, error) {
+
```

Comment:
Can we remove those empty lines after functions or is it more readable that way? It seems inconsistent with the similar code in this project.